app/views/admin/support_notices/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="admin">
+  <!--Descriptive page name, messages and instructions-->
+  <h2 class="heading"><%= t(".page_heading") %></h2>
+  <!--/descriptions-->
+
+  <!--subnav-->
+  <%= render "navigation" %>
+  <!--/subnav-->
+
+  <!--main content-->
+  <ul class="support-notices index group">
+    <%= render "support_notice", support_notice: @support_notice %>
+  </ul>
```

Comment:
The `support-notices` class name reads strange, could you poke either the front end standards or sarken about it? I think it should at least be singular but maybe the dash is also wrong